src/licensedcode/data/rules/other-permissive_208.yml

```diff
@@ -0,0 +1,3 @@
+license_expression: other-permissive
+is_license_text: yes
+relevance: 99
```

Comment:
Please add a `notes: ` field with some link to where such an old license can be found